data/brands.csv

```diff
@@ -273,3 +273,8 @@ XFVC,XF-AVC S/XF-HEVC S File Format,Canon
 ca3a,CMAF Media Profile – AVS3-P3,T-AI-109.7
 clv1,CMAF Media Profile for MPEG-5 LCEVC,CMAF
 isod,Version of the ISO file format,ISO
+cevb,CMAF EVC Baseline Media Profile,CMAF
+cevm,CMAF EVC Main Media Profile,CMAF
+cmfr,CMAF chunk brand,CMAF
```

Comment:
@haudiobe @cconcolato  is the description of `cmfr` correct? From the spec it seems to be defined as a chunk brand, however I'm a little confused about 4CC choice (sounds like a fragment)